src/platforms/javascript/guides/react/configuration/integrations/react-router.mdx

```diff
@@ -34,6 +34,51 @@ Make sure `Sentry.reactRouterV6Instrumentation` is initialized by your `Sentry.i
 
 </Alert>
 
+### Usage with React Router 6.4 Data API
+_(Available in version 7.21.0 and above)_
```

Comment:
Let's move this above v6?

Reply:
The initialization part above is common for all 6.x versions (`reactRouterV6Instrumentation(...)`). Will have to repeat that info if we do that?

Reply:
Ah fair, than we can keep it here.